operator/src/main/java/oracle/kubernetes/operator/helpers/ClusterPresenceInfo.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator.helpers;
+
+import java.util.Optional;
+
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import oracle.kubernetes.operator.work.Component;
+import oracle.kubernetes.operator.work.Packet;
+import oracle.kubernetes.weblogic.domain.model.ClusterResource;
+
+/**
+ * Operator's mapping between custom resource Domain and runtime details about that domain,
+ * including the scan and the Pods and Services for servers.
+ */
```

Comment:
Operator's mapping between custom resource Domain and runtime details about that domain -> "Operator's mapping between custom resource cluster and runtime details about that cluster."